core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1985,12 +1984,15 @@ public void deleteInternal(RpcContext rpcContext, LockedInodePath inodePath,
               job.setCancelState(PersistJob.CancelState.TO_BE_CANCELED);
             }
           }
-          revisedInodesToDelete.add(new Pair<>(alluxioUriToDelete, inodePairToDelete.getSecond()));
```

Comment:
Interestingly in the old code a new `Pair` is created. This is a side-product save in this PR.